docs/learn/vadalog/04-expressions-operators.md

```diff
@@ -163,33 +162,119 @@ q(X,Y,J) :- a(X), b(Y), J = concat(X,Y).
 @output("q").
 ```
 
-The expected output is:
+Expected output:
 
 ```prolog
-q("prometheux","engine","prometheuxengine").
+q("prometheux", "engine", "prometheuxengine").
 ```
 
-A rule with an assignment using the `string_length` operator has the general
-form, and returns an integer representing the length of the string:
+#### `string_length`
+A rule with `string_length` returns an integer representing the length of the `string`:
 
 ```prolog
 q(K1, K2, Kn, J) :- body, J = string_length(string).
 ```
 
-An example for `string_length`:
+Example:
 
 ```prolog showLineNumbers {2}
 a("prometheux").
 q(X, J) :- a(X), J = string_length(X).
 @output("q").
 ```
 
-The expected output is:
+Expected output:
 
 ```prolog
 q("prometheux", 10).
 ```
 
+#### `to_lower`
+A rule with `to_lower` converts `string` to lowercase:
+
+```prolog
+q(K1, K2, Kn, J) :- body, J = to_lower(string).
+```
+
+Example:
+
+```prolog showLineNumbers {2}
+a("Prometheux").
+q(X, J) :- a(X), J = to_lower(X).
+@output("q").
+```
+
+Expected output:
+
+```prolog
```

Comment:
for future: prolog is the default language -- you only need to add it if you also want to add the `showLineNumbers` or the line highlights